includes/wp-adapter.php

```diff
@@ -303,7 +302,7 @@ function pre_update_cron_option( $new_value, $old_value ) {
 			'args'      => $event_to_add['args'],
 		];
 
-		if ( ! empty( $item['value']['schedule'] ) ) {
+		if ( ! empty( $event_to_add['schedule'] ) ) {
```

Comment:
Also including small fix here. The bug would have been that if somebody called `update_option( 'cron'...` directly and had added new recurring cron jobs, we'll currently end up registering them as one-time jobs instead.

Pretty rare scenario, as this is like the backup-backwards-compatibility later, but yeah good to fix for sure :)

Reply:
Gonna pop this into it's own PR w/ tests actually.